content/tools/blend.md

```diff
@@ -25,6 +26,17 @@ summary: |
   while working with Sass colors.
 ---
 
+{% import 'utility.macros.njk' as utility %}
+
+{% set update = ['Update', utility.datetime('2022-04-04')] | join(' ') %}
+{% callout 'note', update %}
+We've been helping to
+[design and implement](https://github.com/sass/sass/blob/main/proposal/color-4-new-spaces.md)
+wide-gamut color support
+directly in the [Sass language](/sass/).
```

Comment:
I think we link to `/sass/` unless we're talking about something we built for them as a client? I don't know…

Reply:
oh nooooooo. I don't have a good answer. Should we try and figure out a way to merge the two?

Reply:
I'd be curious what @SondraE thinks. My initial leaning might be to leave both in place, but prefer linking to `/work/sass-core/` -- and could even mention on that page that you've been a core contributor since 2016.

Reply:
@mirisuzanne I share Jonny's leaning. And I do think that page should indicate that you've been a core contributor.